src/RoslynPad.Common.UI/ViewModels/MainViewModelBase.cs

```diff
@@ -632,7 +636,7 @@ public bool SearchUsingRegex
 
         private void ClearSearch()
         {
-            SearchText = null;
+            _searchText = null;
```

Comment:
Restore the previous line. Otherwise there won't be a property change notification and the search box won't clear on `ClearSearchCommand`. If it's already `null`, the `if` you'll add at the `SearchText` setter will prevent recurrence.